test/extensions/common/wasm/wasm_vm_test.cc

```diff
@@ -49,7 +49,11 @@ class BaseVmTest : public testing::Test {
   Stats::ScopeSharedPtr scope_;
 };
 
-TEST_F(BaseVmTest, NoRuntime) { EXPECT_EQ(createWasmVm(""), nullptr); }
+TEST_F(BaseVmTest, UnSpecifiedRuntime) {
+  auto wasm_vm = createWasmVm("");
+  EXPECT_TRUE(wasm_vm.get() != nullptr);
```

Comment:
Guarding this test on Windows is not a good idea, since people can use `--define wasm=disabled` in their builds, and it will be eventually enabled on Windows, in which case this test would result in false positive failures.

Maybe guard it on `getDefaultWasmEngineName()`?

Reply:
True. Now it is guarded on `getDefaultWasmEngineName()`.